release/nightly_tests/shuffle/large_scale.yaml

```diff
@@ -0,0 +1,19 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```

Comment:
Rename file to datasets_large_scale.yaml to differentiate from the other shuffle yaml?